GUI/QtQuick/AddressListModel.cpp

```diff
@@ -13,132 +13,158 @@
 #include "BTCNumericTypes.h"
 #include "ColorScheme.h"
 
-namespace {
-   static const QHash<int, QByteArray> kRoles{
-      {QmlAddressListModel::TableDataRole, "tableData"},
-      {QmlAddressListModel::HeadingRole, "heading"},
-      {QmlAddressListModel::FirstColRole, "firstcol"},
-      {QmlAddressListModel::ColorRole, "dataColor"}
-   };
+namespace
+{
+    static const QHash<int, QByteArray> kRoles{
+        {QmlAddressListModel::TableDataRole, "tableData"},
+        {QmlAddressListModel::HeadingRole, "heading"},
+        {QmlAddressListModel::FirstColRole, "firstcol"},
+        {QmlAddressListModel::ColorRole, "dataColor"},
+        {QmlAddressListModel::AddressTypeRole, "addressType"}};
 }
 
-QmlAddressListModel::QmlAddressListModel(const std::shared_ptr<spdlog::logger>& logger
-   , QObject* parent)
-   : QAbstractTableModel(parent), logger_(logger)
-   , header_({ tr("Address"), tr("#Tx"), tr("Balance (BTC)"), tr("Comment") })
-{}
+QmlAddressListModel::QmlAddressListModel(const std::shared_ptr<spdlog::logger> &logger, QObject *parent)
+    : QAbstractTableModel(parent), logger_(logger), header_({tr("Address"), tr("#Tx"), tr("Balance (BTC)"), tr("Comment")})
+{
+}
 
 int QmlAddressListModel::rowCount(const QModelIndex &) const
 {
-   return table_.size() + 1;
+    return table_.size() + 1;
 }
 
 int QmlAddressListModel::columnCount(const QModelIndex &) const
 {
-   return header_.size();
+    return header_.size();
 }
 
-QVariant QmlAddressListModel::data(const QModelIndex& index, int role) const
+QVariant QmlAddressListModel::data(const QModelIndex &index, int role) const
 {
-   switch (role) {
-   case TableDataRole:
-      if (index.row() == 0) {
-         return header_.at(index.column());
-      }
-      else {
-         const int row = index.row() - 1;
-         try {
-            switch (index.column()) {
-            case 0:  return table_.at(row).at(0);
-            case 1:  return QString::number(pendingBalances_.at(addresses_.at(row).id()).balance / BTCNumericTypes::BalanceDivider, 'f', 8);
-            case 2:  return QString::number(pendingBalances_.at(addresses_.at(row).id()).nbTx);
-            case 3:  return table_.at(row).at(1);
-            default: return QString{};
+    try
+    {
+        switch (role)
+        {
+        case TableDataRole:
+            if (index.row() == 0)
+            {
```

Comment:
Also opening bracket for everything except function definition and opening a block should on the same line.
General rule of thumb: try not to reformat the existing code, or otherwise it's hard to see your changes.